content/en/real_user_monitoring/browser/modifying_data_and_context.md

```diff
@@ -577,31 +711,78 @@ window.DD_RUM &&
 
 Follow the [Datadog naming convention][16] for a better correlation of your data across the product.
 
+### Clear global context
+
+You can reset the global context by using `clearGlobalContext`.
+
+{{< tabs >}}
+{{% tab "NPM" %}}
+
+```
+import { datadogRum } from '@datadog/browser-rum';
+
+datadogRum.clearGlobalContext();
+
+// Code example
+datadogRum.clearGlobalContext();
+```
+
+{{% /tab %}}
+{{% tab "CDN async" %}}
+```
+DD_RUM.onReady(function() {
+    DD_RUM.clearGlobalContext();
+})
+
+// Code example
+DD_RUM.onReady(function() {
+    DD_RUM.clearGlobalContext()
+})
+```
+{{% /tab %}}
+{{% tab "CDN sync" %}}
+
+```
+window.DD_RUM &&
+    DD_RUM.clearGlobalContext();
+
+// Code example
+window.DD_RUM &&
+    DD_RUM.clearGlobalContext();
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏remove "code examples" as they are the same as the above